crates/rope/src/chunk.rs

```diff
@@ -536,21 +532,14 @@ impl Iterator for Tabs {
         let tab_offset = self.tabs.trailing_zeros() as usize;
         let chars_mask = (1 << tab_offset) - 1;
         let char_offset = (self.chars & chars_mask).count_ones() as usize;
```

Comment:
This might be bad practice because u128 is `Copy` anyway but I didn't know if it helped convey intent if I made `self.chars` a reference since we don't mutate it here any more